agent/go.mod

```diff
@@ -149,3 +149,5 @@ replace github.com/up9inc/mizu/tap/extensions/http v0.0.0 => ../tap/extensions/h
 replace github.com/up9inc/mizu/tap/extensions/kafka v0.0.0 => ../tap/extensions/kafka
 
 replace github.com/up9inc/mizu/tap/extensions/redis v0.0.0 => ../tap/extensions/redis
+
+replace github.com/up9inc/basenine/client/go v0.0.0-20220326121918-785f3061c8ce => /home/mertyildiran/Documents/up9/basenine/client/go
```

Comment:
I guess it should not be here?

Reply:
Removed with https://github.com/up9inc/mizu/pull/984/commits/6596a22fc0199daee8bb32405cdbddf74b89d328